clientv3/client.go

```diff
@@ -105,6 +105,43 @@ func (c *Client) SetEndpoints(eps ...string) {
 	c.balancer.updateAddrs(eps)
 }
 
+// Sync synchronizes client's endpoint.
```

Comment:
Sync synchronizes client's endpoints with the known endpoints from the etcd membership.